primefaces/src/main/java/org/primefaces/component/inplace/InplaceRenderer.java

```diff
@@ -131,7 +132,12 @@ protected String getLabelToRender(FacesContext context, Inplace inplace) {
             return label;
         }
 
-        String value = ComponentUtils.getValueToRender(context, inplace.getChildren().get(0));
+        UIComponent editor = inplace.getChildren().get(0);
+        String value = ComponentUtils.getValueToRender(context, editor);
+        if (LangUtils.isNotBlank(value) && (editor instanceof Password || "password".equalsIgnoreCase(String.valueOf(editor.getAttributes().get("type"))))) {
+            value = "********";
```

Comment:
Maybe we could add a style class to the component instead and not change the value. With this class you can use CSS to add content (bullets by default I'd say).

Reply:
I could see that just want to make sure the unmasked password is not in the HTML.

Reply:
Maybe a `p-masked` style added to it.

Reply:
Sorry, of course we don't want to render the value in case of a password. We could do that by setting the value to null or blank, but that's a bit of a hack if you ask me.